docs/migrate/migrate-qiskit-alg.rst

```diff
@@ -0,0 +1,97 @@
+Work with updated Qiskit algorithms
```

Comment:
So, just put links in here to updated tutorials, with an explanation?

Reply:
Or are we deleting this section and it's just covered by other updates we're making elsewhere?

Reply:
I think that we can keep the section, it would be quite short, but I think that it will help keep the information organized. 

My idea here is to have an introduction explaining that the qiskit algorithms have been updated to use the primitives. Maybe show how VQE has changed with a very minimal example, and then add the links to the tutorials and blog post we have. 

The VQE example could look like this (taken from one of the algorithm guides I have internally):


<table>
<tr>
<th style="text-align: center;">QuantumInstance VQE</th>
<th style="text-align: center;">Estimator VQE</th>
</tr>
<tr>
<td style="text-align: left;">

```python
from qiskit.algorithms.minimum_eigen_solvers import VQE
from qiskit.utils import QuantumInstance
from qiskit import Aer

qi = QuantumInstance(
    backend=Aer.get_backend('statevector_simulator'))
   
vqe = VQE(ansatz, optimizer, quantum_instance=qi)
result = vqe.compute_minimum_eigenvalue(hamiltonian)
```
</td>
<td style="text-align: left;">

```python
# note change of namespace
from qiskit.algorithms.minimum_eigensolvers import VQE
from qiskit.primitives import Estimator

estimator = Estimator()
    
vqe = VQE(estimator, ansatz, optimizer)
result = vqe.compute_minimum_eigenvalue(hamiltonian)
```

</td>
</tr>
</table>